package/MDAnalysis/coordinates/PDB.py

```diff
@@ -151,68 +131,20 @@
 .. autoclass:: MultiPDBWriter
    :members:
 
-.. _strict:
-
-Biopython (strict) PDB Reader and Writer
-----------------------------------------
 
-The :mod:`PDB` module can make use of Biopython's :mod:`Bio.PDB`
-[Hamelryck2003]_ but replaces the standard PDB file parser with one that uses
-the :class:`MDAnalysis.coordinates.pdb.extensions.SloppyStructureBuilder` to
-cope with very large pdb files as commonly encountered in MD simulations. The
-Biopython-based :class:`PDBReader` has the advantage that it implements the
-`PDB standard`_ rigorously but this comes at the cost of flexibility and
-performance. It is also difficult to write out selections using this
-implementation (:class:`PDBWriter`) and multi frame PDB files are not
-implemented. The Biopython Reader/Writer can be selected when loading data into
-a :class:`~MDAnalysis.core.AtomGroup.Universe` by providing the keyword
-*permissive* = ``False``.
+..deprecated:: 0.14.1
+    Setting the flag "permissive_pdb_reader" in :data:`MDAnalysis.core.flags`
+    (see :ref:`flags-label`) to ``False``::
 
-The Biopython PDB parser :class:`Bio.PDB.PDBParser` is fairly strict and even
-in its own permissive mode (which MDAnalysis employs) typically warns about
-missing element names with a
-:exc:`Bio.PDB.PDBExceptions.PDBConstructionWarning` . Such warnings, however,
-are generally harmless and therefore are filtered (and ignored) by MDAnalysis
-with the help of :func:`warnings.filterwarnings`.
-
-
-Classes
-~~~~~~~
-
-.. autoclass:: PDBReader
-   :members:
-
-.. autoclass:: PDBWriter
-   :members:
+    MDAnalysis.core.flags["permissive_pdb_reader"] = False
 
-References
-----------
-
-.. [Hamelryck2003] Hamelryck, T., Manderick, B. (2003) PDB parser and structure
-                    class implemented in Python. Bioinformatics, 19, 2308-2310.
-                    http://biopython.org
-
-.. _PDB standard: http://www.wwpdb.org/documentation/format32/v3.2.html
-.. _END: http://www.wwpdb.org/documentation/format32/sect11.html#END
+    Now yields a placeholder Primitive PDB Reader/Writer that inherits from
```

Comment:
Just mention that it has no effect anymore. This also means you can remove it from the core flags. If you don't feel like it just mention it in #782 that this can be safely removed now.